includes/Install/Installer.php

```diff
@@ -350,6 +351,39 @@ public function create_refund_table() {
         dbDelta( $sql );
     }
 
+    /**
```

Comment:
you need to create db tables from pro version, coz shipping tracking will be a pro feature.